controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -448,8 +468,18 @@ stream {
     }
 
     server {
+        {{ range $address := $all.Cfg.BindAddressIpv4 }}
+        listen                  {{ $address }}:{{ $udpServer.Port }} udp;
+        {{ else }}
```

Comment:
Correct. Only when no valid specified IPv4 address, this `else `will executed and then all IPv4 addresses will be bound.